gen3-integration-tests/services/sheepdog.py

```diff
@@ -0,0 +1,154 @@
+import copy
```

Comment:
would prefer filename to not be the actual service name

Reply:
I think we should keep the service name since it makes it clear which API we are hitting. As discussed, we should expect a lot of maintenance in test code unlike application code (constant changes are introduced by changes in application behavior as well as due to adding and removing tests frequently), so changing them in case the name changes later is ok. But this avoids additional learning curve and design decisions which are hard to solidify (since we expect a large number of contributors here).

Reply:
@Avantol13 @haraprasadj I have already changed the names and tags. I think merging Sheepdog and Peregrine into one "graph data" or "structured data" feature is fine, since that's also what we did in the SDK. Should we sync on this today so we can move forward?

Reply:
Yeah I guess I'm thinking of this more like service/feature code, where it's never certain we'll get a chance to go back and refactor. I suppose I'm good either way, but I'd prefer similar abstractions as the SDK

Reply:
@paulineribeyre should be ok this way too here since we are using SDK and not hitting the APIs directly, just adding a bit of documentation about sdk should suffice here